include/zephyr/net/wifi.h

```diff
@@ -245,4 +245,131 @@ static inline const char *wifi_link_mode_txt(enum wifi_link_mode link_mode)
 	}
 }
 
+enum wifi_ps {
```

Comment:
For the enums that only contain few different values, it saves some bytes if `__packed` is added. Might be useful if the variable is used in structs in order to make them a bit smaller.

Reply:
> Might be useful if the variable is used in structs in order to make them a bit smaller.

Are you suggesting to use `__packed` for structs here, a bit confused by `variable`?

Reply:
No, I am proposing to do 
```
enum {
....
} __packed;
```